fastlane/docs/Actions.md

```diff
@@ -1167,6 +1167,20 @@ testfairy(
 )
 ```
 
+### [DISTRIBUTOR](https://wifidistribution.com)
+
+Upload your iOS build to [DISTRIBUTOR](https://wifidistribution.com)
+
+```ruby
+distributor(
+  api_token: "...",
+  app_id: "...",
+  ipa: "./build.ipa"
```

Comment:
Let's remove the `ipa` block here, as the user gets this for free when using `gym` 👍